spec/unit/knife/bootstrap_options_spec.rb

```diff
@@ -120,6 +123,8 @@ def compare_property(sym)
       :winrm_codepage,
       :concurrency,
       :winrm_shell,
+      :distro, # Deprecated - remove this when the flag is removed.
+      :template_file, # Deprecated - remove this when the flag is removed.
```

Comment:
these are just supposed to be the options that are unique to knife-windows. I don't think you want these two lines.